src/aoimage.cpp

```diff
@@ -8,15 +8,18 @@ AOImage::AOImage(QWidget *parent, AOApplication *p_ao_app, bool make_static) : Q
 {
   m_parent = parent;
   ao_app = p_ao_app;
-  movie = new QMovie();
   is_static = make_static;
-  connect(movie, &QMovie::frameChanged, [=]{
-    QPixmap f_pixmap = movie->currentPixmap();
-    f_pixmap =
-        f_pixmap.scaled(this->size(), Qt::IgnoreAspectRatio);
-    this->setPixmap(f_pixmap);
-    this->setMask(f_pixmap.mask());
-  });
+  if (!is_static) // Only create the QMovie if we're non-static
+  {
+    movie = new QMovie();
```

Comment:
Wait has this been leaking memory this whole time? This needs to be parented. Bruh moment.
```cpp
movie = new QMovie(this);
```